lib/akamai_rspec/matchers/honour_origin_headers.rb

```diff
@@ -30,7 +30,16 @@ def clean_cc_directives(origin_response, akamai_response)
   origin_cc_directives = origin_response.headers[:cache_control].split(/[, ]+/).to_set
   akamai_cc_directives = akamai_response.headers[:cache_control].split(/[, ]+/).to_set
 
-  origin_cc_directives.delete 'must-revalidate' # as Akamai does no pass it on
+  unwanted_values = %w(
+    must-revalidate
+    stale-if-error
+    stale-while-revalidate
+  )
```

Comment:
For future devs reading this code, consider re-adding the comment explaining why the directives are unwan that Akamai doesn't pass these directives along to clients.

Reply:
The previous comment was `# as Akamai does no pass it on` so I'd say the pull request or comment provides a far more suitable overview of why we're doing what we're doing and _why_ Akamai doesn't pass it on.